utils/travis-build-repo-internal.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -l
```

Comment:
I have used the uploaded RPMs in my https://github.com/lindig/xs-shipyard and can confirm that they are working. 

As for using `-l`: I'd avoid it and use something like this:

```
if test -f /etc/profile.d/opam.sh -a -z "$OPAMROOT"; then
  . /etc/profile.d/opam.sh
fi
```